roles/jupyterhub/tasks/packages.yml

```diff
@@ -1,14 +1,14 @@
 ---
 
 - name: JupyterHub conda dependencies
-  conda: name={{item}} state=present
+  command: conda install name={{item}} state=present
```

Comment:
I don't think these two should change. `conda` is an ansible, same as `pip` below. They are slightly more than aliases to the commands.